jekyll-assets/_includes/contribute.html

```diff
@@ -1,5 +0,0 @@
-<div class="contribute">
```

Comment:
Interestingly, it looks like "contribute" used to _only_ be at the bottom of pages like https://www.raspberrypi.com/documentation/accessories/ but not at the bottom of pages like https://www.raspberrypi.com/documentation/accessories/sd-cards.html
I can't remember why! :wink: 

```
$ git grep -C4 footer.html
jekyll-assets/_layouts/boxes.html-    </div>
jekyll-assets/_layouts/boxes.html-
jekyll-assets/_layouts/boxes.html-    {% include contribute.html %} 
jekyll-assets/_layouts/boxes.html-    {% include copyright.html %} 
jekyll-assets/_layouts/boxes.html:    {% include footer.html %}  
jekyll-assets/_layouts/boxes.html-    {% include search.html %}
jekyll-assets/_layouts/boxes.html-
jekyll-assets/_layouts/boxes.html-  </body>
jekyll-assets/_layouts/boxes.html-</html>
--
jekyll-assets/_layouts/docs.html-      </section>
jekyll-assets/_layouts/docs.html-    </div>
jekyll-assets/_layouts/docs.html-
jekyll-assets/_layouts/docs.html-    {% include copyright.html %} 
jekyll-assets/_layouts/docs.html:    {% include footer.html %}  
jekyll-assets/_layouts/docs.html-    {% include scripts.html %}
jekyll-assets/_layouts/docs.html-    {% include search.html %}
jekyll-assets/_layouts/docs.html-
jekyll-assets/_layouts/docs.html-  </body>
```

Reply:
Weird. Because in the old system, only pages that have a GitHub link _lack_ the contribute disclaimer.

Perhaps we should just get rid of it entirely. I suspect we should instead toss that text into the README.md or CONTRIBUTING.md for this repo (if we need it at all?)

Reply:
:shrug: It's entirely up to you...